src/template.js

```diff
@@ -2086,58 +2086,92 @@ module.exports = {
-                    },
-                    'q-applicant-building-and-street-2': {
-                        type: 'string',
-                        title: 'Building and street line 2',
-                        maxLength: 60,
-                        errorMessage: {
-                            maxLength: 'Second line of address must be less than 60 characters'
-                        }
-                    },
-                    'q-applicant-town-or-city': {
-                        type: 'string',
-                        title: 'Town or city',
-                        maxLength: 32,
-                        errorMessage: {
-                            maxLength: 'Town or city must be 32 characters or less'
-                        }
-                    },
-                    'q-applicant-county': {
-                        type: 'string',
-                        title: 'County (optional)',
-                        maxLength: 32,
-                        errorMessage: {
-                            maxLength: 'County must be 32 characters or less'
-                        }
-                    },
-                    'q-applicant-postcode': {
-                        type: 'string',
-                        title: 'Postcode (optional)',
-                        maxLength: 10,
+                allOf: [
+                    {
+                        title: 'Enter your address',
+                        required: ['q-applicant-building-and-street', 'q-applicant-town-or-city'],
+                        propertyNames: {
+                            enum: [
+                                'q-applicant-building-and-street',
+                                'q-applicant-building-and-street-2',
+                                'q-applicant-town-or-city',
+                                'q-applicant-county',
+                                'q-applicant-postcode'
+                            ]
+                        },
+                        allOf: [
+                            {
+                                properties: {
+                                    'q-applicant-building-and-street': {
+                                        type: 'string',
+                                        title: 'Building and street',
+                                        maxLength: 60,
+                                        errorMessage: {
+                                            maxLength:
+                                                'First line of address must be less than 60 characters'
+                                        }
+                                    }
+                                }
+                            },
+                            {
+                                properties: {
+                                    'q-applicant-building-and-street-2': {
+                                        type: 'string',
+                                        title: 'Building and street line 2',
```

Comment:
has this title always been without and `(optional)`?

Reply:
Yes, because this title actually gets hidden by the UISchema. The second line of the applicants address doesn't show a label at all.

Reply:
Probably should have an `(optional)` though. It's "visually" hidden as far as I'm aware, meaning it'll still be read out by assistive technologies.

Reply:
Add an optional if we think it is needed. Consistency is telling me that this should be done. The upcoming changes that BP is working on should have this field get an `(optional)`, so we may as well add it for completeness right now.

Reply:
Yeah, another reason for it to be programmatically added. Cool, it'll get picked up with Barry's automated "optional" PR then 👍